### ISSUE TITLE
Improving Stack dialog?

### ISSUE THREAD

**Issue by shadrackkibet** [africanmathsinitiative/R-Instat#6310]
I now have some different points, while we are here!

I will check with @dannyparsons is the name of the dialogue and also the R-code.  It currently uses melt and unstack uses the opposite.  This has now (I think) changed to pivot_longer and pivot_wider.

We also used a different function when you are stacking multiple sets of columns   We need to see whether we prefer to keep that function, or move that also to livot_longer?  

There is another change that would be useful for stack.  I think the default was (in melt) and still is (in pivot_longer) to carry all remaining variables in the data frame.  Our default (in the dialogue) is to carry nothing, and then to carry the specified variables if you do that.  @dannyparsons would like an option in the dialogue to "Carry All Columns".  We think that is the default in the function.  If so, then it would become the default here too.

_Originally posted by @rdstern in https://github.com/africanmathsinitiative/R-Instat/issues/5831#issuecomment-656208693_

**Comment by dannyparsons** [comment]
Yes to all of these things. We should change to `pivot_longer` though the syntax for multiple column stacking is more complex so dialog design will take some thought.
